src/main/kotlin/pl/edu/agh/server/domain/user/User.kt

```diff
@@ -64,6 +64,12 @@ class User(
     @OneToMany(mappedBy = "user", cascade = [CascadeType.ALL], orphanRemoval = true, fetch = FetchType.LAZY)
     var organizationUserRoles: MutableList<OrganizationUserRole> = mutableListOf()
 
+    var verificationToken: String? = null
```

Comment:
Troche edge case, ale z tego co widze, jak uzytkownik sie zarejestruje i przed walidacja zmieni sobie haslo, to token mu sie nadpisze i juz nigdy nie bedzie mogl potwierdzic konta :D